en/developer_guide/multiPanel.rst

```diff
@@ -0,0 +1,83 @@
+Add panel to an administration view
+===================================
+
+Context
+-------
+
+When working with administration back office, sometimes you need to add your own forms or contents to the existing one.
+Open Orchestra implements a system to easily add form to your administration page in a new panel.
+
+Panel System
+~~~~~~~~~~~~
+
+Each administration facade have a list of links used by ajax to load their form.
+To display a new panel, add a link to the facade, prefixed by ``_self_panel``.
+
+.. code-block:: php
+
+    $linkName = "_self_panel_" + $position + "_" + $panelTitle;
+    $facade->addLink($linkName, $route);
+
+The link name is a key to find your link in an array.
+Open Orchestra also use this key to choose position and name of your panel:
```

Comment:
uses
the position
"an array" => what array ? If it's useful to talk about it it should be clear what is meant